spec/rubocop/cop/rspec/expect_in_hook_spec.rb

```diff
@@ -76,4 +76,22 @@
       end
     RUBY
   end
+
```

Comment:
[This](https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/rspec/shared_contexts.rb#L42) might be used somehow to stub `AllCops`. I believe we'll have to patch it in the long run to accept arbitrary keypairs (that would also allow moving Rails-related code to `rspec-rails`).
It's ok to use a copy of it now.

This doesn't solve the issue of `ALL` being pre-calculated though.
What if we take a non-performant way and define `ALL` as a method that would return `BUILD` plus fetch from config?

I vote for postponing the decision on how to cache config and be flexible in the specs at the same time.